adapters/grid/grid.go

```
@@ -312,6 +359,9 @@ func (a *GridAdapter) MakeRequests(request *openrtb2.BidRequest, reqInfo *adapte
 	request.Imp = validImps
 
 	reqJSON, err := json.Marshal(request)
+
+	fixedReqJSON, err := fixNative(reqJSON)
```

Please do not ignore error from the previous statement